src/pages/costModels/costModel/updateMarkupDialog.tsx

```diff
@@ -62,76 +91,111 @@ class UpdateMarkupModelBase extends React.Component<Props, State> {
+                this.state.sign === this.state.origSign) ||
               isLoading
             }
           >
-            {t('cost_models_details.add_rate_modal.save')}
+            {t('save')}
           </Button>,
           <Button
             key="cancel"
             variant="link"
             onClick={() => onClose({ name: 'updateMarkup', isOpen: false })}
             isDisabled={isLoading}
           >
-            {t('cost_models_details.add_rate_modal.cancel')}
+            {t('cancel')}
           </Button>,
         ]}
       >
         <Stack hasGutter>
           <StackItem>{error && <Alert variant="danger" title={`${error}`} />}</StackItem>
-          <StackItem>
-            <Title headingLevel="h2" size="md">
-              {t('cost_models_details.table.columns.name')}
-            </Title>
-          </StackItem>
           <StackItem>
             <TextContent>
-              <Text component={TextVariants.h6}>{current.name}</Text>
+              <Text style={styles.cardDescription}>
+                {t('cost_models_details.description_markup_or_discount_model')}
+              </Text>
             </TextContent>
           </StackItem>
           <StackItem>
             <Form>
-              <FormGroup
-                label={t('cost_models_wizard.markup.markup_label')}
-                fieldId="markup-input-box"
-                helperTextInvalid={t('cost_models_wizard.markup.invalid_markup_text')}
-                validated={!isNaN(Number(this.state.markup)) ? 'default' : 'error'}
-              >
-                <InputGroup style={{ width: '150px' }}>
-                  <TextInput
-                    type="text"
-                    aria-label={t('cost_models_wizard.markup.markup_label')}
-                    id="markup-input-box"
-                    value={this.state.markup}
-                    onChange={(markup: string) => {
-                      const markupDecimal = Number(markup);
-                      const dx = markup.split('').findIndex(c => c === '.');
-                      if (!isNaN(markupDecimal) && dx > -1 && markup.length - dx - 1 > 2) {
-                        this.setState({
-                          markup: formatValue(markupDecimal, 'markup', {
-                            fractionDigits: 2,
-                          }) as string,
-                        });
-                        return;
-                      }
-                      this.setState({ markup });
-                    }}
-                    validated={!isNaN(Number(this.state.markup)) ? 'default' : 'error'}
-                  />
-                  <InputGroupText style={{ borderLeft: '0' }}>%</InputGroupText>
-                </InputGroup>
-              </FormGroup>
+              <Grid hasGutter>
+                <GridItem lg={8} id="refSign">
+                  <FormGroup isInline fieldId="markup-or-discount" label={t('cost_models_details.markup_or_discount')}>
+                    <div style={styles.radioAlign}>
```

Comment:
Don't use a margin to push the radio button down, align it vertically; using center or bottom.

Note that we don't need to place each radio button in an extra `div` to apply a style. `GridItem`, `FormGroup`, and `radio` accept a class name.